nautobot/dcim/filters.py

```diff
@@ -230,9 +224,14 @@ class Meta:
 
 
 class RackFilterSet(NautobotFilterSet, TenancyFilterSet, StatusModelFilterSetMixin):
-    q = django_filters.CharFilter(
-        method="search",
-        label="Search",
+    q = SearchFilter(
+        filter_predicates={
+            "name": "icontains",
+            "facility_id": "icontains",
+            "serial": "icontains",
+            "asset_tag": "icontains",
```

Comment:
Let's see if we want to "reintroduce" value stripping on these fields.

Reply:
FINE. I did it.